<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationDlg</class>
 <widget class="QDialog" name="AnimationDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>605</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>605</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="bt_back">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>674</y>
     <width>34</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-image: url(:/Images/AnimationControl/leftNormal.tif);
}
QPushButton::hover{
	border-image: url(:/Images/AnimationControl/leftHover.tif);
}
QPushButton::Pressed{
	border-image: url(:/Images/AnimationControl/leftPress.tif);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btCancel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>750</y>
     <width>111</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image:url(:/Images/Btn_Gray.png);
}
QPushButton:hover{
	border-image:url(:/Images/Btn_Gray.png);
	color:green;
}
QPushButton:pressed{
	border-image: url(:/Images/Btn_Gray_Pressed.png);
	color:white;
}</string>
   </property>
   <property name="text">
    <string>ResStr_Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_next">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>674</y>
     <width>34</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{	
	border-image: url(:/Images/AnimationControl/rightNormal.tif);
}
QPushButton::hover{
	border-image: url(:/Images/AnimationControl/rightHover.tif);
}
QPushButton::Pressed{
	border-image: url(:/Images/AnimationControl/rightPress.tif);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>720</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ResStr_Prompt_Box_will_not_pop_up_next_time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_msg1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>560</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ResStr_Prompt1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_pause_play">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>670</y>
     <width>44</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{	
	border-image: url(:/Images/AnimationControl/pauseNormal.tif);
}
QPushButton::hover{	
	border-image: url(:/Images/AnimationControl/pauseHover.tif);
}
QPushButton::Pressed{
	border-image: url(:/Images/AnimationControl/pausePress.tif);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_msg2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>600</y>
     <width>605</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>ResStr_Prompt2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btOK">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>750</y>
     <width>111</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-image:url(:/Images/Btn_Green.png);
	color: rgb(255, 255, 255);
}
QPushButton:hover{
	border-image:url(:/Images/Btn_Green.png);
	color:brack;
}
QPushButton:pressed{
border-image:url(:/Images/Btn_Green_Pressed.png);
color: rgb(255, 255, 255);
}
</string>
   </property>
   <property name="text">
    <string>ResStr_OK</string>
   </property>
  </widget>
  <zorder>bt_back</zorder>
  <zorder>btCancel</zorder>
  <zorder>bt_next</zorder>
  <zorder>radioButton</zorder>
  <zorder>label_msg1</zorder>
  <zorder>bt_pause_play</zorder>
  <zorder>btOK</zorder>
  <zorder>label_msg2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
